components/dashboard/src/teams/git-integrations/GitIntegrationModal.tsx

```diff
@@ -289,6 +297,8 @@ const getPlaceholderForIntegrationType = (type: ProviderType) => {
             return "github.example.com";
```

Comment:
As a follow-up, I'd be interested to understand how we could shift this logic to the server side and send the info to the instance, such that we don't need to hard-code those providers.